schemas/cs3/tbl/magic_bo.ksy

```diff
@@ -0,0 +1,53 @@
+meta:
+  id: magic_bo
+  endian: le
+doc: |
+  Effects and parameter of Brave Order count effects.
+
+  Present in t_magic.tbl.
+seq:
+  - id: skill_id
+    type: u2
+  - id: bo_effect_id_1
+    type: u2
+  - id: bo_effect_param_1_1
+    type: s4
+  - id: bo_effect_param_2_1
+    type: s4
+  - id: bo_effect_param_3_1
+    type: s4
+  - id: bo_effect_id_2
+    type: u2
+  - id: bo_effect_param_1_2
+    type: s4
+  - id: bo_effect_param_2_2
+    type: s4
+  - id: bo_effect_param_3_2
+    type: s4
+  - id: bo_effect_id_3
+    type: u2
+  - id: bo_effect_param_1_3
+    type: s4
+  - id: bo_effect_param_2_3
+    type: s4
+  - id: bo_effect_param_3_3
+    type: s4
+  - id: bo_effect_id_4
+    type: u2
+  - id: bo_effect_param_1_4
+    type: s4
+  - id: bo_effect_param_2_4
+    type: s4
+  - id: bo_effect_param_3_4
+    type: s4
+  - id: bo_effect_id_4
+    type: u2
+  - id: bo_effect_param_1_4
+    type: s4
+  - id: bo_effect_param_2_4
+    type: s4
+  - id: bo_effect_param_3_4

```

Comment:
> Are the brave order effect ids shared (or free of overlap) with effect ids for items and magic? If they are we can re-use the effect schema from outstanding PR #3. Otherwise we should make a new one and get rid of the repetition with a `repeat-expr`.

Yes, they are.

Reply:
In that case we can use this: https://github.com/nnguyen259/SenSchema/pull/2/files#diff-c33004666ec918acd7c999cb577491cefd56b5ffdf42b69a299b3108e8924ad7R76-R79 plus an import of `/components/cs3/effects` in the `meta` section.